README.md

```diff
@@ -120,6 +120,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
+- [Read more about custom instrumentation you can manually enable](docs/custom_instrumentation.md)
+- [Read more about configuring the SDK with authentication](docs/auth.md)
```

Comment:
I think the auth docs need to be rebased off of https://github.com/aws-observability/aws-otel-swift/pull/89